lib/chat_api/customers/customer.ex

```diff
@@ -102,4 +102,10 @@ defmodule ChatApi.Customers.Customer do
       :time_zone
     ])
   end
+
+  def test_changeset(message, attrs) do
+    message
```

Comment:
nit -- let's change `message` to `customer`

Reply:
Argh copy and paste error. Good catch thanks!